### PR TITLE
compute overlap using width x2-x1 width

### PR DESCRIPTION
**Pull request by brunoeducsantos**
Hi,
I just want rectify the extra +1 in computation of width and height in overlap evaluation.
Let me know if it makes sense.

Regards,
Bruno

### CODE REVIEW

File: keras_retinanet/utils/compute_overlap.pyx [hunk 1]
```diff
@@ -30,23 +30,23 @@ def compute_overlap(
     cdef unsigned int k, n
     for k in range(K):
         box_area = (
-            (query_boxes[k, 2] - query_boxes[k, 0] + 1) *
-            (query_boxes[k, 3] - query_boxes[k, 1] + 1)
+            (query_boxes[k, 2] - query_boxes[k, 0] ) *
```
Comment (hgaiser) [inline keras_retinanet/utils/compute_overlap.pyx:33]: Could you remove the extra space before the parenthesis here?

File: keras_retinanet/utils/compute_overlap.pyx [hunk 2]
```diff
@@ -30,23 +30,23 @@ def compute_overlap(
...
                 ih = (
                     min(boxes[n, 3], query_boxes[k, 3]) -
-                    max(boxes[n, 1], query_boxes[k, 1]) + 1
+                    max(boxes[n, 1], query_boxes[k, 1]) 
                 )
                 if ih > 0:
                     ua = np.float64(
-                        (boxes[n, 2] - boxes[n, 0] + 1) *
-                        (boxes[n, 3] - boxes[n, 1] + 1) +
+                        (boxes[n, 2] - boxes[n, 0] ) *
```
Comment (hgaiser) [inline keras_retinanet/utils/compute_overlap.pyx:48]: and here

**Comment by brunoeducsantos** [comment]
Thanks for the feedback. Spaces removed

**Comment by brunoeducsantos** [comment]
Hi @hgaiser ,
Could I be added to the list of contributors?
Thanks a lot,
Bruno

**Comment by hgaiser** [comment]
Could you make a PR for that ? ^^

**Comment by brunoeducsantos** [comment]
Sorry, I didnt understand. What do you mean a PR for that?

**Comment by hgaiser** [comment]
Usually people add themselves to https://github.com/fizyr/keras-retinanet/blob/master/CONTRIBUTORS.md in a PR.

**Comment by bw4sz** [comment]
I just confirmed that the following fails on latest pypi release (OSX), but is solved by this pull request. Somewhat high priority. 

```
(test) MacBook-Pro:keras-retinanet ben$ python
Python 3.7.6 | packaged by conda-forge | (default, Jun  1 2020, 18:33:30)
[Clang 9.0.1 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from keras_retinanet.utils.anchors import compute_overlap
Using TensorFlow backend.
>>> import numpy as np
>>>
>>> true_array = np.expand_dims(np.array([0.,0.,5.,5.]),axis=0)
>>> prediction_array = np.expand_dims(np.array([0.,0.,10.,10.]), axis=0)
>>> retinanet_iou = compute_overlap(prediction_array,true_array)
>>> assert retinanet_iou[0][0] == (5**2/10**2)
```

**Pull request merged**